src/main/java/com/school/project/controller/UserController.java

```diff
@@ -21,9 +23,30 @@
     @Autowired
     UserService userService;
 
+    @Autowired
+    UserAccountService userAccountService;
+
     @PostMapping("/users")
-    public void createUser(@RequestBody UserDto userDto) {
-        userService.create(convertUserDtoToUser(userDto));
+    public ResponseEntity<UserDto> createUser(@RequestBody UserDto userDto) {
+        return ResponseEntity.ok()
+                .body(convertUserToUserDto(userService
+                        .create(convertUserDtoToUser(userDto))));
+    }
+
+    @PutMapping("/users/userAccounts/{id}")
+    public ResponseEntity updateUserAccountType(@PathVariable long id, @RequestBody int accountType){
+        userAccountService.getUserAccountByUserId(id).setAccountRole(chooseType(accountType));
+        return ResponseEntity.ok().build();
+    }
+
+    private UserAccountType chooseType(int accountType){
```

Comment:
why is it in Controller?? Please remember, controller doesn't manipulate data. You should do everything in service. And why do you need this function? Enum knows it's id : https://docs.oracle.com/javase/8/docs/api/java/lang/Enum.html